frontend/src/components/pages/ManageUserPage.tsx

```diff
@@ -0,0 +1,135 @@
+import { Table } from "react-bootstrap";
+import { Role } from "../../types/AuthTypes";
+import UserAPIClient from "../../APIClients/UserAPIClient";
+import { User } from "../../types/UserTypes";
+
+const ManageUser = (): React.ReactElement => {
+  const [role, setRole] = useState<Role>("Administrator");
+  const [users, setUsers] = useState<User[]>([]);
+  const [page, setPage] = useState(1);
+  const [usersPerPage, setUsersPerPage] = useState(10);
+  useEffect(() => {
+    async function getUsers() {
+      const fetchedUsers = await UserAPIClient.getUsersByRole(role);
+      setUsers(fetchedUsers);
+      setPage(0);
+    }
+    getUsers();
+  }, [role]);
+
+  const emptyRows =
+    page > 0 ? Math.max(0, (1 + page) * usersPerPage - users.length) : 0;
+
+  const handleChangePage = (
+    event: React.MouseEvent<HTMLButtonElement> | null,
+    newPage: number,
+  ) => {
+    setPage(newPage);
+  };
+
+  const handleChangeRowsPerPage = (
+    event: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>,
+  ) => {
+    setUsersPerPage(parseInt(event.target.value, 10));
+    setPage(0);
+  };
+
+  return (
+    <Stack direction="column" justifyContent="center" margin="2rem" spacing={2}>
+      <FormControl fullWidth>
+        <InputLabel id="role-select-label">Role</InputLabel>
+        <Select
+          labelId="role-select-label"
+          id="role-select"
+          value={role}
+          label="Role"
+          onChange={(e: SelectChangeEvent<Role>) =>
+            setRole(e.target.value as Role)
+          }
+        >
+          <MenuItem value="Administrator">Administrator</MenuItem>
+          <MenuItem value="Facilitator">Facilitator</MenuItem>
+          <MenuItem value="Learner">Learner</MenuItem>
+        </Select>
+      </FormControl>
+      <TableContainer component={Paper}>
+        <Table aria-label="User grouped by role table">
+          <TableHead>
+            <TableRow>
+              <TableCell>Email</TableCell>
+              <TableCell align="right">First Name</TableCell>
+              <TableCell align="right">Last Name</TableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {(usersPerPage > 0
+              ? users.slice(
+                  page * usersPerPage,
+                  page * usersPerPage + usersPerPage,
+                )
+              : users
+            ).map((user) => (
+              <TableRow key={user.id}>
+                <TableCell component="th" scope="row">
+                  {user.email}
+                </TableCell>
+                <TableCell style={{ width: 160 }} align="right">
+                  {user.firstName}
+                </TableCell>
+                <TableCell style={{ width: 160 }} align="right">
+                  {user.lastName}
+                </TableCell>
+              </TableRow>
+            ))}
+            {emptyRows > 0 && (
+              <TableRow style={{ height: 25 * emptyRows }}>
```

Comment:
this'll make one tall empty row - did you mean to make several empty rows, like a `TableRow` for each empty row?

Reply:
The I just copied from this https://mui.com/material-ui/react-table/

Does it make more sense to create 1 big row or multiple rows to fill out the empty space?

Reply:
Ohhh ok I get what you mean... you're right, 1 big empty space would be better.

Was the whole `emptyRows` thing also something you got from MUI docs? I was looking at https://mui.com/material-ui/react-table/#data-table and it seems like the empty space comes with the component already

Reply:
I think I copied the one under custom pagination options. Should I use that one or the one you linked?

Reply:
Hmm we can leave it like this for now, and then once designers make designs for this page we can revisit if needed. What do you think?

I remember noticing a change in height between page 1 and page 2 of Facilitators @ 5 per page when I was testing last night, so I wasn't sure if it's a bug... but if that's how MUI had it we can leave it at that and polish when we implement designs. All good as far as this ticket goes either way! Thanks 😎

Reply:
ok so basically the data table looked too scary and I didn't want to do it the way they did it so I just copied how it was done in the customize seciton. I changed it so every row is equal height and stuff